docs/remote_inference_blueprints/bedrock_connector_blueprint.md

```diff
@@ -0,0 +1,106 @@
+### Bedrock connector blueprint example
+
+1. Add connector endpoint to trusted URLs:
+
+```json
+PUT /_cluster/settings
+{
+    "persistent": {
+        "plugins.ml_commons.trusted_connector_endpoints_regex": [
+            "^https://bedrock-runtime\\..*[a-z0-9-]\\.amazonaws\\.com/.*$"
+        ]
+    }
+}
+```
+
+2. Create connector for Amazon Bedrock:
+
+```json
+POST /_plugins/_ml/model_groups/_register
+{
+    "name": "Amazon Bedrock",
+    "description": "Test connector for Amazon Bedrock",
+    "version": 1,
+    "protocol": "aws_sigv4",
+    "credential": {
+        "roleArn": "<PLEASE ADD YOUR AWS ROLE ARN HERE>"
```

Comment:
roleArn is not supported in open source

Reply:
@ylwu-amzn  - I expanded to add an example for both. Tested working on self-managed 2.10 with AWS credentials.